packages/@jsii/benchmarks/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as Benchmark from 'benchmark';
+import * as cp from 'child_process';
+import * as fs from 'fs-extra';
+import { Compiler } from 'jsii/lib/compiler';
+import { loadProjectInfo } from 'jsii/lib/project-info';
+import * as os from 'os';
+import * as path from 'path';
+import { promisify } from 'util';
+
+// Always run against the same version of CDK source
+const CDK_TAG = 'v2.21.1';
+const exec = promisify(cp.exec);
+async function setupAwsCdkLib(): Promise<{ awsCdkLib: string }> {
+  const repoDir = await fs.mkdtemp(path.join(os.tmpdir(), 'jsii-cdk-bench'));
+  await exec(`git clone -b ${CDK_TAG} https://github.com/aws/aws-cdk.git .`, {
```

Comment:
A shallow clone will be much faster. Can also consider downloading the release's source ZIP file from GitHub instead...

```suggestion
  await exec(`git clone --depth=1 -b ${CDK_TAG} https://github.com/aws/aws-cdk.git .`, {
```